### PR TITLE
Config prettier eslint

### PR DESCRIPTION
**Pull request by IkePureza**
This configures prettier eslint to the project:

- Adds eslint-config-prettier to make ESLint and Prettier play nice with each other
- Adds empty prettier config file so that editors know we are using prettier 
- Adds git pre commit hook to run prettier
- Formats all files with prettier

### CODE REVIEW

File: app/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (zlatax) [inline app/.prettierrc.json:1]: Does this mean use the default prettier config?

Comment (IkePureza) [reply app/.prettierrc.json:1]: Yep

**Review by zlatax — APPROVED** [review]
Looks good!

**Comment by IkePureza** [comment]
Alredy done as part of ID011

**Pull request closed without merging**